src/main/java/pl/noip/lolstats/lol/stats/controller/RegistrationController.java

```diff
@@ -0,0 +1,23 @@
+package pl.noip.lolstats.lol.stats.controller;
+
+import org.springframework.web.bind.annotation.*;
+import pl.noip.lolstats.lol.stats.dto.RegistrationRequest;
+import pl.noip.lolstats.lol.stats.dto.Repository;
+
+@RestController
+@RequestMapping("/register")
+public class RegistrationController {
+
+    private Repository repository;
+
+    public RegistrationController(Repository repository) {
+        this.repository = repository;
+    }
+
+    @PostMapping
+    public void register(@RequestBody RegistrationRequest registrationReguest) {
+        System.out.println(registrationReguest.getEmail());
```

Comment:
System.out.prinltn shouldnt be present in code, You can use it for debugging, but not in production code